Add function to manage label on the PR

punkymaniac opened the pull request.

punkymaniac committed:
Add function to manage label on the PR

gbonnefille commented:
Thanks for this contribution.

Can you write a minor introduction in the README?

Concerning the naming convention, I started by adding the name `tuleap` in the name of all functions as it seems to be a global scope. Perhaps can we rename this function as `labelTuleapPR`?

punkymaniac committed:
Rename files labelPR to labelTuleapPR

punkymaniac committed:
Add labelTuleapPR description to README

punkymaniac committed:
Fix variable definition

punkymaniac committed:
Change invalid script name on test script

The script labelPR.grrovy as been renomed as labelTuleapPR.groovy.
The change has been forgotten in the test script.
